scripts/xsslint/xsslint/linters.py

```diff
@@ -457,15 +457,13 @@ def _check_javascript_escape(self, file_contents, results):
         """
         Checks that only necessary escape() are used.
```

Comment:
This sentence is vague and unclear.  I know it is old.  You introduced the phrase "dangerous escape functions" below.  Maybe that could be added to the docstring, explaining what the dangerous escape functions are?

Reply:
Agreed. I have updated that and my added comment to clarify things.